centos/deprecated.md

```diff
@@ -0,0 +1 @@
+This image is no longer supported/maintained (last updated September 15, 2021, [docker-library/official-images#10304](https://github.com/docker-library/official-images/pull/10304); see also https://www.centos.org/centos-linux-eol/). Please adjust your usage accordingly.
```

Comment:
Should this point to the earlier PRs that actually changed `7.x` (https://github.com/docker-library/official-images/pull/9082 and https://github.com/docker-library/official-images/pull/9102 for `ppc64le` and https://github.com/docker-library/official-images/pull/5146 for `arm32v7` :see_no_evil:)?

Reply:
Yeah, it's a bit complicated!  This PR is the last time the image maintainer updated any tag of the image, but there's one slightly newer where you removed the EOL versions (which is what the PR I linked to was updating) where these are the most recent PRs to update an ostensibly "supported" version. :weary:

Maybe we should change the wording to make it clear this is the last time the maintainer updated any part of the image, not necessarily the supported tags?  Maybe we should instead drop this bit of the message entirely?  Not sure.

Reply:
Ok, updated with some alternative wording for your consideration. :bow: